modules/services/dropbox.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.dropbox;
+  baseDir = ".dropbox-hm";
+  dropboxCmd = ''
+    ${pkgs.coreutils}/bin/env -i HOME="$HOME" ${pkgs.dropbox-cli}/bin/dropbox'';
+  homeBaseDir = "${config.home.homeDirectory}/${baseDir}";
+in {
+  meta.maintainers = [ maintainers.eyjhb ];
+
+  options = {
+    services.dropbox = {
+      enable = mkEnableOption "Dropbox daemon";
+
+      path = mkOption {
+        type = types.path;
+        default = "${config.home.homeDirectory}/Dropbox";
+        defaultText = "\${config.home.homeDirectory}/Dropbox";
+        description = "Where to put the Dropbox directory.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = [ pkgs.dropbox-cli ];
+
+    systemd.user.services.dropbox = {
+      Unit = { Description = "Starting dropbox"; };
```

Comment:
This seems like an odd description, is the "Starting" part of the Dropbox name? Seeing messages like "Starting Starting dropbox", "Started Starting dropbox", "Stopped Starting dropbox", etc. in the journal might be confusing.

Reply:
Fixed in next push